/**:
  ros__parameters:
    # -- algorithm --
    lookahead_distance_ratio: 2.2
    min_lookahead_distance: 2.5
    reverse_min_lookahead_distance: 7.0
    converged_steer_rad: 0.1
